plumbing/msg/sender.go

```diff
@@ -56,7 +76,7 @@ func (s *Sender) Send(ctx context.Context, from, to address.Address, value *type
 	s.l.Lock()
 	defer s.l.Unlock()
 
-	st, err := s.chainReader.LatestState(ctx)
+	st, err := s.chainState.LatestState(ctx)
```

Comment:
from the Naming Is Hard Department: I hope that we intend to converge on one convention for a chain.ReadStore -- I've seen chainreader, chain, chainstore, and now chainstate.  This isn't a request for changes here; maybe it's something we can discuss next week.

Reply:
This one I renamed because I inserted a scoped interface with this name (that happens to be satisfied by the chain store).